lib/widgets/lightbox.dart

```diff
@@ -583,3 +586,43 @@ Route<void> getLightboxRoute({
     },
   );
 }
+
+Route<void> getImageLightboxRoute({
+  int? accountId,
+  BuildContext? context,
+  required Message message,
+  required Uri src,
+  required Uri? thumbnailUrl,
```

Comment:
nit: include trailing comma in intermediate commit

(that helps reduce the diff both for that commit and the one after it)